backend/infrahub/computed_attribute/tasks.py

```diff
@@ -463,51 +469,21 @@ async def computed_attribute_setup(branch_name: str | None = None) -> None:  # p
     flow_run_name="Setup computed attributes for Python transforms in task-manager",
 )
 async def computed_attribute_setup_python(
-    branch_name: str | None = None,  # pylint: disable=unused-argument
+    branch_name: str | None = None, commit: str | None = None, trigger_updates: bool = True
 ) -> None:
     log = get_run_logger()
     service = services.service
 
-    if branch_name:
-        await add_tags(branches=[branch_name])
-
-    await wait_for_schema_to_converge(branch_name=registry.default_branch, service=service, log=log)
-
-    schema_branch = registry.schema.get_schema_branch(name=registry.default_branch)
-
-    transform_attributes = schema_branch.computed_attributes.python_attributes_by_transform
+    branch_name = branch_name or registry.default_branch
 
-    transform_names = list(transform_attributes.keys())
+    await add_tags(branches=[branch_name])
 
-    transforms = await service.client.filters(
-        kind="CoreTransformPython",
-        branch=registry.default_branch,
-        prefetch_relationships=True,
-        populate_store=True,
-        name__values=transform_names,
-    )
+    await wait_for_schema_to_converge(branch_name=branch_name, service=service, log=log)
+    if commit:
+        print(f"Processing commit {commit} - {branch_name}")
+        log.info(f"Processing commit {commit} - {branch_name}")
```

Comment:
do we need both?

Reply:
No that part shouldn't have been there at all it was just while testing. The commit will be used later but not now. I've removed this if-statement completely.